Makefile

```diff
@@ -150,8 +153,14 @@ timebomb-check:
 	        false; \
 	    fi
 
+# Given the path to a file containing 'sha245:<image id>' return <image id>
```

Comment:
Not worth a repush on its own

Reply:
Oh whups.  I can fix that NP, this PR builds quickly.